cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -268,6 +269,41 @@ def validate_class_has_properties_or_parent(self) -> Self:
             )
         return self
 
+    @model_serializer(mode="plain", when_used="always")
+    def information_rules_serializer(self, info: SerializationInfo) -> dict[str, Any]:
+        kwargs = vars(info)
+        default_prefix = f"{self.metadata.prefix}:" if self.metadata.prefix else ""
+
+        field_names = ["Class"] if info.by_alias else ["class_"]
+        properties = []
+        for prop in self.properties:
+            dumped = prop.model_dump(**kwargs)
+            for field_name in field_names:
+                if value := dumped.get(field_name):
+                    dumped[field_name] = value.removeprefix(default_prefix)
+            properties.append(dumped)
+
+        field_names = ["Class"] if info.by_alias else ["class_"]
+        classes = []
+        parent_name = "Parent Class" if info.by_alias else "parent"
+        for prop in self.properties:
```

Comment:
I think this is error, you should iterate over `self.classes` not `self.properties`

Reply:
You are right